integration-tests/e2e-tests/package.json

```diff
@@ -21,22 +20,21 @@
   },
   "homepage": "https://github.com/amagyar-iohk/atala-prism-wallet-sdk-ts-e2e#readme",
   "dependencies": {
-    "@cucumber/cucumber": "^9.3.0",
+    "@cucumber/cucumber": "^9.6.0",
+    "@input-output-hk/atala-prism-wallet-sdk": "^2.4.1",
```

Comment:
I removed this because it was installing the already packaged version instead of the tar.gz created in the previous step
This shouldn't be necessary?

Reply:
it would be added again when the release is made, we have a daily job which uses the last version. during PR this dependency is replaced by the local package